src/components/base/BaseButton.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <button type="button" class="btn">
+    <slot></slot>
+  </button>
+</template>
```

Comment:
Nitpick, but I'd encourage having component related styles in the component file for easier readability or at least include their styles file in the component.